policyengine_us/variables/gov/states/id/tax/income/deductions/id_aged_or_disabled_deduction.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class id_aged_or_disabled_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho aged or disabled deduction"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        age = person("age", period)
+        head = person("is_tax_unit_head", period)
+        spouse = person("is_tax_unit_spouse", period)
+        disabled = person("is_disabled", period)
```

Comment:
we need to include the clause for disabled people that more than one half of their support comes from the filer 

we can recreate the files from the following PR: `care_and_support_costs` `care_and_support_payment`
https://github.com/PolicyEngine/policyengine-us/pull/2384/files
(just copy the files with the same metadata

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/06652d5f-728a-4268-9cb7-6dfa9fab4f55)